neuron-deploy/docker-example/Dockerfile-libmode

```diff
@@ -14,23 +14,24 @@ RUN apt-get update -y \
     wget \
     python3-pip \
     python3-setuptools \
-    && cd /usr/local/bin \
-    && pip3 --no-cache-dir install --upgrade pip \
-    && rm -rf /var/lib/apt/lists/* \
-    && apt-get clean
+    && pip3 --no-cache-dir install --upgrade pip
 
 RUN echo "deb https://apt.repos.neuron.amazonaws.com bionic main" > /etc/apt/sources.list.d/neuron.list
-RUN wget -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB | apt-key add -
+RUN wget --no-check-certificate -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB | apt-key add -
```

Comment:
We don't recommend this for on [host installation](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/neuron-intro/pytorch-setup/pytorch-install.html).  Maybe in the comment we can add if we see any issue with ssl certificates then add --no-check-certificate

Reply:
@aws-vrnatham Sounds good! Changed it to a comment instead.